tardis/simulation/base.py

```diff
@@ -415,14 +416,16 @@ def log_plasma_state(
 
         plasma_state_log = pd.DataFrame(
             index=np.arange(len(t_rad)),
-            columns=["t_rad", "next_t_rad", "w", "next_w"],
+            columns=["t_rad (K)", "next_t_rad (K)", "w", "next_w"],
+        )
+        plasma_state_log["t_rad (K)"] = np.around(t_rad.value, decimals=3)
```

Comment:
why are you rounding things there.

Reply:
This variable is a `np.ndarray` which consists of floating values. There were two ways which allowed to have the significant digits in any precision. 
- One of them was `np.around()` which rounds up the value to the decimals that would be passed via the parameter. 
- Another, one is the `np.format_float_positional()` but this is performance intensive in terms of calculation & will increase the runtime. These two were the most viable options to have the desired result on the `ndarray` without any other code change. 

One another way could be to create a function, which will change the decimal precision by dividing the values for each item in the `ndarray` for the appropriate decimal but this may increase the runtime in unexpected ways. 

I will check if something better can be done to have the same results 😉

Reply:
Removed the rounding done to the value. Values are now automatically truncated to 3 decimal when displaying the dataframe for plasma values ie `t_rad` and `w`.